templates/default/mongod.conf.erb

```diff
@@ -0,0 +1,17 @@
+# mongodb config file written by Chef
```

Comment:
```suggestion
# This file was generated by Chef Infra
# Do NOT modify this file by hand.
```

Reply:
I will add that; is this a rule for all templates? (If so, should I add it to the Chef style guide?)